Client/Frontend/Settings/Clearables.swift

```diff
@@ -126,24 +115,14 @@ class CacheClearable: Clearable {
             // Remove the basic cache.
             URLCache.shared.removeAllCachedResponses()
             
-            var err: Error?
-            for item in ["Caches", "Cookies", "WebKit"] {
-                do {
-                    try deleteLibraryFolderContents(item, validateClearedExceptFor: ["Snapshots"])
-                } catch {
-                    err = error
-                }
+            WKWebsiteDataStore.default().removeData(ofTypes: WKWebsiteDataStore.allWebsiteDataTypes(), modifiedSince: Date(timeIntervalSinceReferenceDate: 0)) {
+                // Clear image cache
```

Comment:
This comment can be removed, it adds nothing of value, I know it was there before the changes :)

Reply:
Removed :)